crates/ty_python_semantic/resources/mdtest/named_tuple.md

```diff
@@ -150,9 +150,10 @@ class Person(NamedTuple):
 
 reveal_type(Person._field_defaults)  # revealed: dict[str, Any]
 reveal_type(Person._fields)  # revealed: tuple[str, ...]
-reveal_type(Person._make)  # revealed: bound method <class 'Person'>._make(iterable: Iterable[Any]) -> Self
+# revealed: bound method <class 'Person'>._make[Self: NamedTupleFallback](iterable: Iterable[Any]) -> Self
```

Comment:
Yeah, I think we should skip `Self` when printing functions. Conceptually `Self` is really (implicitly) scoped to the class, not to the function.